discovery-provider/plugins/network-monitoring/src/discovery/queries.ts

```diff
@@ -128,13 +129,28 @@ const _importContentNodes = async (run_id: number) => {
     preferHigherPatchForSecondaries: true,
   });
   await audiusLibs.init();
-  const contentNodes = await audiusLibs.ServiceProvider?.listCreatorNodes();
+  // const contentNodes = await audiusLibs.ServiceProvider?.listCreatorNodes();
+  const numberOfProviders = await audiusLibs.ethContracts?.ServiceProviderFactoryClient.getTotalServiceTypeProviders(
+    "content-node"
+  ) || 0
```

Comment:
why default to 0? should prob just throw on failure, or retry

Reply:
takes 1min to add retries, but yeh sure - can see. agree failures are v unlikely